src/gatekeeper.conf

```diff
@@ -8,7 +8,8 @@ enable-refresh-tokens: true
 enable-logging: true
 preserve-host: true
 secure-cookie: true # SHOULD BE true!! Only set this to false for local dev
-enable-logging: true
+server-write-timeout: 1h # Timeout fetching complete response
+upstream-response-header-timeout: 1h  # Time until header is received
```

Comment:
Om aan te sluiten bij de openstack config, en om dump ook goed te laten werken staat de timeout op 30000 secs. Misschien is het handig om alle timeouts dezelfde waarde ter laten houden

Reply:
Ah ik zat met 1h in m'n hoofd.

Ik neem aan dat je 3.000 bedoelt, en geen 30.000?

Reply:
Hm toch 30.000 in Openstack idd. Aangepast.